.gitignore

```diff
@@ -4,6 +4,7 @@ node_modules
 # Hardhat files
 /cache
 /artifacts
+/vibcArtifacts
```

Comment:
This would create breaking changes in the PR for updated logic to retreive ABIs. What was your rationale for adding this?

Reply:
I saw your earlier attempts to get the ABIs and the issue is that while we depend on vibc-core it does not compile everything in there.

Which is why I believed it would be better to compile vibc at installation, and store the resulting ABIs (the full ones, not just interfaces) in this folder.